be/src/block_cache/block_cache.cpp

```diff
@@ -83,45 +83,49 @@ Status BlockCache::init(const CacheOptions& options) {
     return _kv_cache->init(options);
 }
 
-Status BlockCache::write_cache(const CacheKey& cache_key, off_t offset, const IOBuffer& buffer, size_t ttl_seconds,
-                               bool overwrite) {
+Status BlockCache::write_cache(const CacheKey& cache_key, off_t offset, const IOBuffer& buffer,
+                               WriteCacheOptions* options) {
```

Comment:
Normally, `XxxOptions` should be const reference instead of pointer

Reply:
> Normally, `XxxOptions` should be const reference instead of pointer

Use a pointer because it is an optional parameter and is nullptr if we not set. Also, some output metric options will be add to this options, we we does not restrict it with `const`.